plugins/vpp/ipsecplugin/vppcalls/dump_vppcalls.go

```diff
@@ -199,17 +204,14 @@ func (h *IPSecVppHandler) DumpIPSecSPD() (spdList []*IPSecSpdDetails, err error)
 		SpdMeta: make(map[string]*SpdMeta),
 	}
 
-	// TODO IPSec SPD dump request requires SPD ID, otherwise it returns nothing. There is currently no way
-	// to dump all SPDs available on the VPP, so let's dump at least the ones configurator knows about.
-	for _, spdName := range h.spdIndexes.GetMapping().ListNames() {
-		spdIdx, _, found := h.spdIndexes.LookupIdx(spdName)
-		if !found {
-			// Shouldn't happen, call the police or something
-			continue
-		}
+	// Get all VPP SPD indexes
+	spdIndexes, err := h.dumpSpdIndexes()
+	if err != nil {
+		return nil, errors.Errorf("failed to dump SPD indexes: %v", err)
+	}
+	for spdIdx, numPolitics := range spdIndexes {
```

Comment:
It's **policies**, not politics. Politics are in government.

Reply:
that's a typo, thanks for the correction 😄